vispy/visuals/image.py

```diff
@@ -116,36 +138,64 @@ class ImageVisual(Visual):
     if the data are 2D.
     """
     def __init__(self, data=None, method='auto', grid=(1, 1),
-                 cmap='cubehelix', clim='auto', **kwargs):
+                 cmap='cubehelix', clim='auto',
+                 interpolation='Nearest', **kwargs):
         self._data = None
-        self._interpolation = 'nearest'
+
+        # check texture interpolation
+        self._interpolation = interpolation
+        if self._interpolation == 'Bilinear':
+            texture_interpolation = 'linear'
+        else:
+            texture_interpolation = 'nearest'
```

Comment:
can you check to make sure that the `interpolation` is in the tuple of accepted values? Also, for consistency we should probably have them be all lower case, e.g. `spline16` instead of `Spline16`